From 2b859b1fd398540139f2f3d336474a0c141c5d4c Mon Sep 17 00:00:00 2001
From: Marc Mutz <marc.mutz@qt.io>
Date: Thu, 4 Sep 2025 14:46:53 +0200
Subject: [PATCH 001/553] [docs] Fix links to QTest::setThrowOnFail()

The functions are called _set_ThrowOnX, not throwOnX. Fix their
spelling in the QTEST_THROW_ON_{FAIL,SKIP} macros to enable linking to
the resp. function.

Amends e769cf026e328ed7fff660c204ce6e55b80114e3.

Pick-to: 6.9 6.8
Change-Id: I40c47ceabcdc36019aa4f1029b06b7040ab5c001
Reviewed-by: Edward Welbourne <edward.welbourne@qt.io>
(cherry picked from commit 1dddb00cb3999d90350f954dc6ef0108c2f14829)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/testlib/qtestcase.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/testlib/qtestcase.cpp b/src/testlib/qtestcase.cpp
index 53cc849e23c..22c8fc5581d 100644
--- a/src/testlib/qtestcase.cpp
+++ b/src/testlib/qtestcase.cpp
@@ -210,7 +210,7 @@ void Internal::maybeThrowOnSkip()
     \relates <QTest>
 
     When defined, QCOMPARE()/QVERIFY() etc always throw on failure.
-    QTest::throwOnFail() then no longer has any effect.
+    QTest::setThrowOnFail() then no longer has any effect.
 */
 
 /*!
@@ -218,7 +218,7 @@ void Internal::maybeThrowOnSkip()
     \macro QTEST_THROW_ON_SKIP
     \relates <QTest>
 
-    When defined, QSKIP() always throws. QTest::throwOnSkip() then no longer
+    When defined, QSKIP() always throws. QTest::setThrowOnSkip() then no longer
     has any effect.
 */
 
-- 
2.51.2

